modules/sdk-coin-eth2/test/unit/eth2.ts

```diff
@@ -50,17 +49,19 @@ describe('Ethereum 2.0', function () {
     this.skip();
     const localBaseCoin = bitgo.coin('teth2');
     const keyPair = localBaseCoin.generateKeyPair();
-    keyPair.pub.length.should.equal(98);
-    keyPair.secretShares.every((secretShare) => secretShare.length.should.equal(66));
-    (keyPair.pub.startsWith('0x')).should.be.true();
-    keyPair.secretShares.every((secretShare) => (secretShare.startsWith('0x')).should.be.true());
-    localBaseCoin.isValidPub(keyPair.pub).should.be.true();
+    keyPair.pub?.length.should.equal(98);
+    // keyPair.secretShares.every((secretShare) => secretShare.length.should.equal(66));
+    keyPair.pub?.startsWith('0x').should.be.true();
+    // keyPair.secretShares.every((secretShare) => secretShare.startsWith('0x').should.be.true());
+    localBaseCoin.isValidPub(keyPair.pub!).should.be.true();
```

Comment:
What is the purpose of adding `!`?

Reply:
stricter type enforcements in coin modules. It's now just cast `as string` instead of using the non-null assertion.